#Skin temperature message
Header header
float32 st